src/ReactHover.js

```diff
@@ -90,7 +95,7 @@ class ReactHover extends Component {
     if (isNaN(shiftY)) {
       shiftY = 0
     }
-    updatedStyles = { ...hoverComponentStyle, top: cursorY + shiftY, left: cursorX + shiftX }
+    updatedStyles = { ...hoverComponentStyle, top: cursorY + shiftY - domPos.y, left: cursorX + shiftX - domPos.x }
```

Comment:
According to https://caniuse.com/#search=getBoundingClientRect, the return value for `getBoundingClientRect` lacks `x` and `y` for ie and edge, we can use `left` and `top` to cover these 2 browsers.